generate/convert.go

```diff
@@ -171,7 +171,11 @@ func (g *generator) convertArguments(
 			return nil, err
 		}
 
-		goName := upperFirst(arg.Variable)
+		goName := arg.Variable
+		if g.Config.AutoCamelCase {
+			goName = snakeToCamel(goName)
+		}
+		goName = upperFirst(goName)
```

Comment:
may be worth breaking this into a method now? especially since that will make it easy to add support for `default: raw` as I describe in the config